lib/miq_automation_engine/models/miq_ae_datastore.rb

```diff
@@ -169,10 +169,28 @@ def self.reset_to_defaults
       reset_domain(DATASTORE_DIRECTORY, domain_name, Tenant.root_tenant)
     end
 
+    reset_domains_from_provider_gems
+
     restore_attrs_for_domains(saved_attrs)
     reset_default_namespace
   end
 
+  def self.reset_domains_from_provider_gems
+    provider_gems = Vmdb::Plugins.instance.vmdb_plugins
+    provider_gems.each do |provider_gem|
+      domain_bucket = provider_gem.root.join("db", "fixtures", "ae_datastore")
+      Dir.glob(domain_bucket.join("*")).each do |domain_location|
+        next unless File.directory?(domain_location)
```

Comment:
@bdunne 
do we need db/fixtures/ae_datastore/<<domains>> could we just have ae_datastore/<<domains>?